README.md

```diff
@@ -4,7 +4,7 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/25b4a4111f831121dda5/test_coverage)](https://codeclimate.com/github/sul-dlss/mais_orcid_client/test_coverage)
 
 # mais_orcid_client
-API client for accessing MAIS's ORCID endpoints.
+API client for accessing MAIS's ORCID endpoints, e.g. using MAIS to check ORCID permissions and getting the access token for ORCID.
```

Comment:
This is only some of the methods supported by this gem.

Reply:
I would be happy to add an "or" clause, or change this to say something like "to push/pull from orcid, use this client for XXX and use sul_orcid_client for xxx".

Reply:
How about something like:
> MAIS's ORCID API provides access to ORCID information for Stanford users. (This is different from orcid.org's ORCID API, which is supported by https://github.com/sul-dlss/orcid_client.)

Reply:
took your suggestion.